client/src/components/ui/Select.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    export let options: Array<string> = [];
+    export let index = 0;
+    export let value;
```

Comment:
What's the data type for this? We should also initialize it to `null` so that we won't be surprised if it's uninitialized.

Reply:
Right now, the use case in `Sandbox` its a string. However, if the need arises it may need to carry a different datatype altogether. For not lets make it a string.
```suggestion
    export let value: string = '';
```